automation/orchestrated_pr_runner.py

```
@@ -0,0 +1,385 @@
+            log(f"Spawned agent for {repo_full}#{pr_number} at /tmp/{repo}/{workspace_name}")
+            success = True
+        else:
+            log(f"Failed to spawn agent for {repo_full}#{pr_number}")
+    return success
+
+
+def run_fixpr_batch(cutoff_hours: int = DEFAULT_CUTOFF_HOURS, max_prs: int = DEFAULT_MAX_PRS) -> None:
+    log(f"Discovering open PRs updated in last {cutoff_hours}h for org {ORG}")
+    try:
+        prs = query_recent_prs(cutoff_hours)
+    except Exception as exc:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)